lms/djangoapps/teams/views.py

```diff
@@ -267,6 +278,13 @@ def get(self, request):
             )
 
         if 'topic_id' in request.QUERY_PARAMS:
+            topic_id = request.QUERY_PARAMS['topic_id']
+            if topic_id not in [topic['id'] for topic in course_module.teams_configuration['topics']]:
+                error = build_api_error(
+                    ugettext_noop('The supplied topic id {topic_id} is not valid'),
```

Comment:
Is this message user-visible? I'm can't remember when/where in the workflow someone might enter a topic ID.